Tasks/UsePythonVersionV0/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -8,6 +8,8 @@
   "loc.input.help.versionSpec": "Version range or exact version of a Python version to use, using semver's version range syntax. [More information](https://go.microsoft.com/fwlink/?LinkID=2006180)",
   "loc.input.label.allowUnstable": "Allow downloading unstable releases",
   "loc.input.help.allowUnstable": "Allow downloading unstable python versions from [Github Actions python versions registry](https://github.com/actions/python-versions).",
+  "loc.input.label.githubToken": "GitHub token",
```

Comment:
Could 'GitHub token for Github Actions python registry' be more relevant here?

Reply:
Fixed